ngraph/python/src/pyngraph/frontend/frontend.cpp

```diff
@@ -19,10 +19,11 @@ void regclass_pyngraph_FrontEnd(py::module m)
         m, "FrontEnd", py::dynamic_attr());
     fem.doc() = "ngraph.impl.FrontEnd wraps ngraph::frontend::FrontEnd";
 
-    fem.def("load_from_file",
-            &ngraph::frontend::FrontEnd::load_from_file,
-            py::arg("path"),
-            R"(
+    fem.def(
+        "load",
+        [](ngraph::frontend::FrontEnd& self, const std::string& s) { self.load(s); },
```

Comment:
RETURN self.load(s)!!!
Please run tests manually as on CI not all environment is set up (there are tickets for that)